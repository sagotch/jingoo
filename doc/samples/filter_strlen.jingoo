<dl>
  <dd>strlen("hogehoge")</dd>
  <dt>{{ strlen("hogehoge") }}</dt>
  <dt>strlen("日本語")</dt>
  <dd>{{ strlen("日本語") }}</dd>
</dl>
