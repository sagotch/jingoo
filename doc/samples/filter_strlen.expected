<dl>
  <dd>strlen("hogehoge")</dd>
  <dt>8</dt>
  <dt>strlen("日本語")</dt>
  <dd>3</dd>
</dl>
